src/pages/addCard/components/addCard/CardPasswordInput.tsx

```diff
@@ -0,0 +1,44 @@
+import { Input, InputProps } from '@components/ui-kit';
+import { Dot } from '@pages/addCard/components';
+
+const INPUT_WIDTH = '40px';
+const INPUT_MAX_LENGTH = 1;
+
+interface CardPasswordInputProps {
+	firstValue: string;
+	secondValue: string;
+	onFirstValueChange: InputProps['onChange'];
+	onSecondValueChange: InputProps['onChange'];
+}
+
+export default function CardPasswordInput({
+	firstValue,
+	secondValue,
+	onFirstValueChange,
+	onSecondValueChange,
+}: CardPasswordInputProps) {
+	return (
+		<Input.Wrapper>
+			<Input.Label label="카드 비밀번호" />
+			<Input
+				variant="basic"
+				value={firstValue}
+				onChange={onFirstValueChange}
+				type="password"
+				maxLength={INPUT_MAX_LENGTH}
+				width={INPUT_WIDTH}
+			>
+				<Input
+					variant="basic"
+					value={secondValue}
+					onChange={onSecondValueChange}
+					type="password"
+					maxLength={INPUT_MAX_LENGTH}
+					width={INPUT_WIDTH}
+				/>
+				<Dot width={INPUT_WIDTH} color="#94dacd" />
+				<Dot width={INPUT_WIDTH} color="#94dacd" />
+			</Input>
+		</Input.Wrapper>
```

Comment:
```suggestion
interface CardPasswordInputProps {
  firstProps: Pick<InputProps, 'value' | 'onChange'>;
  secondProps: Pick<InputProps, 'value' | 'onChange'>;
}

export default function CardPasswordInput({
	firstProps,
	secondProps,
}: CardPasswordInputProps) {
	return (
		<Input.Wrapper>
			<Input.Label label="카드 비밀번호" />
			<Input
			  {...firstProps}
				variant="basic"
				type="password"
				maxLength={INPUT_MAX_LENGTH}
				width={INPUT_WIDTH}
			>
				<Input
  			  {...secondProps}
					variant="basic"
					type="password"
					maxLength={INPUT_MAX_LENGTH}
					width={INPUT_WIDTH}
				/>
				<Dot width={INPUT_WIDTH} color="#94dacd" />
				<Dot width={INPUT_WIDTH} color="#94dacd" />
			</Input>
		</Input.Wrapper>
```

이러면 더 유연하게 표현할 수 있지 않을까 싶네요!